azure_functions_worker/utils/app_setting_manager.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+import os
+
+from ..constants import (PYTHON_ROLLBACK_CWD_PATH,
+                         PYTHON_THREADPOOL_THREAD_COUNT,
+                         PYTHON_ISOLATE_WORKER_DEPENDENCIES,
+                         PYTHON_ENABLE_WORKER_EXTENSIONS,
+                         PYTHON_ENABLE_WORKER_EXTENSIONS_DEFAULT,
+                         PYTHON_ENABLE_WORKER_EXTENSIONS_DEFAULT_39,
+                         PYTHON_ENABLE_DEBUG_LOGGING,
+                         FUNCTIONS_WORKER_SHARED_MEMORY_DATA_TRANSFER_ENABLED)
+
+
+def get_python_appsetting_state():
+    current_vars = os.environ.copy()
+    python_specific_settings = \
+        [PYTHON_ROLLBACK_CWD_PATH,
+         PYTHON_THREADPOOL_THREAD_COUNT,
+         PYTHON_ISOLATE_WORKER_DEPENDENCIES,
+         PYTHON_ENABLE_DEBUG_LOGGING,
+         FUNCTIONS_WORKER_SHARED_MEMORY_DATA_TRANSFER_ENABLED]
+
+    app_setting_states = ""
+
+    for app_setting in python_specific_settings:
+        if app_setting in current_vars:
+            app_setting_states += (app_setting + ": "
+                                   + current_vars[app_setting] + " ")
+
+    # Special case for extensions
+    python_version = current_vars['FUNCTIONS_WORKER_RUNTIME_VERSION']
+    if python_version == '3.9':
+        app_setting_states += (PYTHON_ENABLE_WORKER_EXTENSIONS + ": "
```

Comment:
This is not checking if the customer has enabled this or not.

Reply:
Or are you only printing the defaults? In this case how do we check if the customer has enabled it or not?

Reply:
Added it to the list of variables to check and log. If it's not an env variable, we log the default value